uploadNeuroDB/mass_pic.pl

```diff
@@ -131,3 +163,27 @@
 
 print "\nFinished mass_pic.pl execution\n" if $verbose;
 exit 0;
+
+1;
```

Comment:
is this `1` intentionally there?

Reply:
tested it, and it works fine, but I still think this 1 should not be here!

Reply:
oups... Probably copied it from the redmine ticket by mistake. Nice catch!!